src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul class="stars--0">
+      <li class="star--0--1"></li>
+      <li class="star--0--2"></li>
+      <li class="star--0--3"></li>
+      <li class="star--0--4"></li>
+      <li class="star--0--5"></li>
+    </ul>
```

Comment:
Use `div` instead of `ul>li`
You dont need modifier on each star. Use `:nth-child`

```suggestion
    <div class="stars stars--0">
      <div class="stars__star" />
      <div class="stars__star" />
      <div class="stars__star" />
      <div class="stars__star" />
      <div class="stars__star" />
    </div >
```